### ISSUE TITLE
[Feature] Expanded side Menu option

### ISSUE THREAD

**Issue by SJ1O** [jurialmunkey/skin.arctic.fuse#265]
### Skin section

Home Widgets

### Feature description

Back  on version 3.25 it was possible to have the side menu expand so could see the text labels on the side menu items next to the icons. Would it be possible to re enable this in the latest version.

### Why is this feature needed? Please justify and explain.

I personally think that it give the menu navigation a cleaner and more user friendly experience.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Duplicate of #241

See my comment here on reason why it had to be removed for the time being:
https://forum.kodi.tv/showthread.php?tid=373859&pid=3172868#pid3172868

> I was forced to remove it because it didn't work well when there was more than five side items.
>
> The labels were separate from the menu to avoid expanding its hitbox (problem for mouse/touch on widgets near the menu) but separate labels are statically positioned and so won't scroll up/down with the menu.
>

**Comment by SJ1O** [comment]
Ahh thanks for clearing that up and explaining why it was removed. Was a nice feature and hopefully one day it returns even if it only functions when you have 5 or less menu items :) it's all very complex tho so I understand if it doesn't return no big deal. I also agree with what others are saying I definitely seem to see some speed improvements so that's a plus. And I also went back to the old version like another guy asked because he was struggling to set it up I have a backup of version V3.25 and original reverted back to that as I was struggling to set up my widgets but I give it another go last night and thankfully got it set up. However I did struggle for a while just got abit confused with the widget options and didn't understand how to actually edit the hub but all good now glad I got it sorted. Thanks for everything it's much appreciated.
